pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/FieldMergeRule.scala

```diff
@@ -45,20 +46,31 @@ trait FieldMergeRule {
   protected trait PartialRule {
     val isDefinedForTarget: WorkPredicate
     val isDefinedForSource: WorkPredicate
+    val isDefinedForSourceList: Seq[TransformedBaseWork] => Boolean =
```

Comment:
For a rule to be applied:
* The `target` has to return `true` with the `isDefinedForTarget` predicate
* The `sources` has to return `true` with the `isDefinedForSourceList` predicate
* Only is some `sources` that return `true` with the `isDefinedForSource` predicate

And only the `isDefinedForSource` `sources` get sent to the rule.

Is that about right?

Reply:
That is exactly right